src/spdigger_stack.c

```diff
@@ -2192,9 +2191,23 @@ struct Thing *check_place_to_pickup_spell(struct Thing *thing, long a2, long a3)
     return _DK_check_place_to_pickup_spell(thing, a2, a3);
 }
 
-struct Thing *check_place_to_pickup_unconscious_body(struct Thing *thing, long a2, long a3)
+struct Thing *check_place_to_pickup_unconscious_body(struct Thing *thing, MapSubtlCoord stl_x, MapSubtlCoord stl_y)
 {
-    return _DK_check_place_to_pickup_unconscious_body(thing, a2, a3);
+    // return _DK_check_place_to_pickup_unconscious_body(thing, a2, a3);
+    struct Map* mapblk = get_map_block_at(stl_x, stl_y);
+    struct Thing *result = thing_get(get_mapwho_thing_index(mapblk));
+    if (thing_is_invalid(result))
+      return INVALID_THING;
+    while ( result->class_id != TCls_Creature
```

Comment:
use instead !thing_is_creature(thing)

Reply:
Why? That'll break it: what if the first thing on the map block is valid, but not a creature?

Reply:
Or do you mean in the while statement? If so, I don't see the point, seeing as there's already a check to see if it's invalid right before it.

Reply:
!thing_is_creature(thing) is a more readable way to say result->class_id != TCls_Creature